aeron-agent/src/main/java/io/aeron/agent/ArchiveEventDissector.java

```diff
@@ -515,6 +515,106 @@ static void dissectReplicationSessionDone(
             .append(" isSynced=").append(isSynced);
     }
 
+    static void dissectReplaySessionStateChange(
+        final MutableDirectBuffer buffer, final int offset, final StringBuilder builder)
+    {
+        int absoluteOffset = offset;
+        absoluteOffset += dissectLogHeader(CONTEXT, REPLAY_SESSION_STATE_CHANGE, buffer, absoluteOffset, builder);
+
+        final long sessionId = buffer.getLong(absoluteOffset, LITTLE_ENDIAN);
+        absoluteOffset += SIZE_OF_LONG;
+        final long recordingId = buffer.getLong(absoluteOffset, LITTLE_ENDIAN);
+        absoluteOffset += SIZE_OF_LONG;
+        final long position = buffer.getLong(absoluteOffset, LITTLE_ENDIAN);
+        absoluteOffset += SIZE_OF_LONG;
+
+        builder.append(": sessionId=").append(sessionId);
```

Comment:
This should be `": replaySessionId="`.
What would be also valuable is to print the `sessionId` of the replay Publication which is [encoded in the lower 32 bits](https://github.com/real-logic/aeron/blob/master/aeron-archive/src/main/java/io/aeron/archive/ArchiveConductor.java#L782) of the `replaySessionId`. 


For example:
```
replaySessionId=16231259302 sessionId=-948609882
```
or
```
replaySessionId=16231259302 replayId=3 sessionId=-948609882
```